opentracing-impl-java8/src/test/java/io/opentracing/propagation/TestTextMapExtractorImpl.java

```diff
@@ -13,17 +13,16 @@
  */
 package io.opentracing.propagation;
 
+import io.opentracing.SpanContext;
+import io.opentracing.TestSpanContextImpl;
+
 import java.util.Iterator;
 import java.util.Map;
 
-import io.opentracing.propagation.TextMapReader;
-import io.opentracing.SpanContext;
-
-public class TestTextMapExtractorImpl implements Extractor {
-    public SpanContext extract(Object carrier) {
-        TextMapReader textMapCarrier = (TextMapReader)carrier;
```

Comment:
per https://github.com/opentracing/opentracing-java/pull/33/files#r71980119 , note how the right side of the diff can just declare the `extract(TextMap carrier)` directly, eliminating the need for this cast.